lib/matplotlib/axis.py

```diff
@@ -1078,11 +1078,10 @@ def get_tightbbox(self, renderer):
             if (np.isfinite(bbox.width) and np.isfinite(bbox.height) and
                     a.get_visible()):
                 bb.append(bbox)
-
         bb.extend(ticklabelBoxes)
         bb.extend(ticklabelBoxes2)
-
-        bb = [b for b in bb if b.width != 0 or b.height != 0]
+        bb = [b for b in bb if ((b.width != 0 or b.height != 0) and
+                                np.isfinite(b.x0 + b.y0 + b.x1 + b.y1))]
```

Comment:
np.isfinite(bbox.width) and np.isfinite(bbox.height) ? (not necessarily clearer, but consistent with 6 lines above)

Reply:
I think this also takes care of cases where the bb has moved off to negative infinity, but I'm not 100% sure why that happens now...

Reply:
As above...